clients/mobile/.env.example

```diff
@@ -0,0 +1,13 @@
+COUNTRY=
```

Comment:
don't leave things empty
give examples of the kind and format of values that needs setting